### DISCUSSION TITLE
please delete this Post

### DISCUSSION THREAD

**Discussion by DinVision** [valinet/ExplorerPatcher#3729] [Ideas]
{sorry, admin, please delete this Post}

**Comment by pyrates999** [comment]
Read the release notes on what directories and files you need to exclude with your AV: https://github.com/valinet/ExplorerPatcher/releases

also read about EP's implementation of the windows 10 taskbar here: https://github.com/valinet/ExplorerPatcher/wiki/ExplorerPatcher's-taskbar-implementation

**Comment by Ruskie-it** [comment]
I have the same issue, unfortunately the "purple right arrow" corporate has blocked EP and this very same windows layout is now impossible to obtain.
Directories can't be excluded from scan by normal users (and even if we could, it would be a violation of company policies. :( )

**Comment by pyrates999** [comment]
You should be discussing this with your Manager at work to allow you to install EP then.

**Reply by Ruskie-it** [reply]
Absolutely. In some case it might be even possible, but not with such big corporation as mine is. I even tried to write something, actually, but it's a rubber wall. They don't listen to employees, not even if something would boost their productivity

**Reply by pyrates999** [reply]
I wish you the best of luck then.
